docs/ru/easytofly.md

```diff
@@ -0,0 +1,142 @@
+## Информация о команде
+Название команды: EasyToFly
+
+В команде 5 человек: 
+- Игорь Сидорин [@maerans12](https://github.com/maerans12) (TeamLead)
+- Артём Баталов [@bart02](https://github.com/bart02) (Full-Stack разработчик)
+- Карина Янышевская [@fanot](https://github.com/fanot) (Веб-разработчик)
+- Никита Локтев [@nikilokser](https://github.com/nikilokser) (Специалист отдела Hard)
+- Даниил Руфин (Специалист отдела Hard)
+
+## Введение
+
+В работе рассматривается процесс разработки образовательного аппаратно-программного комплекса (далее – АПК, система) для безопасного обучения специалистов по профилю "Летающая робототехника".
+
+Основной **проблемой данной сферы** является недостаточная безопасность открытых систем беспилотных летательных аппаратов (далее – БПЛА), а также сложность в их управлении и программировании для использования в обучении, промышленности и т.д.
+
+**Объектом проектной работы** являются учебные комплексы для изучения процессов автоматизации. **Предметом** является АПК для безопасного обучения специалистов по профилю "Летающая робототехника".
+
+**Цель**: Разработка АПК для простого и безопасного обучения специалистов по профилю "Летающая робототехника".
+
+**Задачи**:
+
+1. Проведение исследования среди потенциальных потребителей об их необходимостях при обучении специалистов
+2. Проведение анализа существующих решений
+3. Разработка технического задания (далее – ТЗ) на АПК
+4. Изучение литературы, необходимой для реализации проекта
+5. Формирование команды исполнителей на основании технического задания
+6. Разработка программной подсистемы АПК «Предотвращение столкновений»
+7. Разработка программной подсистемы АПК «Мастер первоначальной настройки БПЛА»
+8. Разработка программной подсистемы АПК «Монитор состояния БПЛА»
+9. Разработка аппаратной составляющей АПК
+
+![Лого](https://user-images.githubusercontent.com/19249148/89198142-edf20400-d5d6-11ea-9a83-c4ec9c989313.png)
```

Comment:
Please, commit images to the repo. External images are not allowed.

Reply:
https://github.com/CopterExpress/clover/pull/309/commits/f505bc5ca311624a68d606ece5e43ffb45b9ceee